add-on/chrome/modules/LoopSidebar.jsm

```diff
@@ -38,6 +38,13 @@ var LoopSidebarInternal = {
    */
   createSidebar: function(window, token) {
     let ownerDocument = window.gBrowser.ownerDocument;
+
+    // We shouldn't ever create more than one sidebar, so just protect against
+    // it here.
+    if (ownerDocument.getElementById("loop-side-browser")) {
+      return;
+    }
+
```

Comment:
Probably worth documenting this behavior in the JSDoc.  I'd suggest also making the MozLoopAPI.createSidebar JSDoc just point to this instead of having a second copy of the doc that needs to be kept in sync.